docs/installation_cosmo.rst

```diff
@@ -8,12 +8,31 @@ To keep it light, maintainable and easily extensible, **cobaya** does not includ
 Installing a basic set of codes and likelihoods
 -----------------------------------------------
 
-To install a comprehensive set of cosmology requisites (CAMB, CLASS, Planck, BICEP-Keck, BAO, SN), in a ``/path/to/packages`` folders of your choice:
+To install a basic set of cosmology requisites (CAMB, CLASS, Planck, BAO, SN), in a ``/path/to/packages`` folders of your choice:
 
 .. code:: bash
 
    $ cobaya-install cosmo -p /path/to/packages
 
+This does not install the Planck 2018 **clik** likelihoods, the original packages can then be installed using e.g.
```

Comment:
Regarding this block, I think it is not a good approach not to install by default at least one "full" high-ell TTTEEE likelihood. If we do not want clik by default, given how problematic it is, I suggest we add the NPIPE likelihoods to the basic install. The way it is now, `cobaya-install cosmo` is insufficient to run full Planck (PR3 or PR4).

Reply:
I did wonder about that. Can add planck_NPIPE_highl_CamSpec.TTTEEE to install_basic. But the NPIPE lensing is a separate package, so would still not have consistent NPIPE setup and still need to separately run cobaya-install on the actual yaml being used to get the lensing installed. (moving forward I was thinking could encourage people to run cobaya-install on actual yaml, then more things can be external packages)

An alternative would be to add native plik_lite.